### PR TITLE
Adds TF2 layer tests witn new FE in precommit

### PR DESCRIPTION
**Pull request by rnugmanx**
### Details:
 - Marks necessary tests to run in precommit 
 - Adds configurations in config yml

### Tickets:
 - CVS-93285

### CODE REVIEW

File: .ci/azure/linux.yml [hunk 1]
```diff
@@ -455,30 +455,39 @@ jobs:
...
+      continueOnError: false
+
+    - script: |
+        python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
+        export PYTHONPATH=$(REPO_DIR)/tools/mo/:$(LAYER_TESTS_DIR):$PYTHONPATH
+        export OV_FRONTEND_PATH=$(INSTALL_DIR)/runtime/lib/intel64
+        export TEST_DEVICE=CPU
+        $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/tensorflow2_keras_tests/ --use_new_frontend -m precommit_tf_fe --junitxml=$(INSTALL_TEST_DIR)/TEST-tf2_keras_fe.xmlTEST
+      displayName: 'TensorFlow 2 Layer Tests - New FE'
+      continueOnError: false
```
Comment (rkazants) [inline .ci/azure/linux.yml:480]: Please put layer tests for TF FE first and legacy after. Please align the names of these tests with the current version

**Review by rkazants — CHANGES_REQUESTED** [review]

File: .ci/azure/linux.yml (hunk 1 above)
Comment (rkazants) [reply .ci/azure/linux.yml:480]: `TensorFlow 2 Layer Tests - New FE` -> `TensorFlow 2 Layer Tests - TF FE`

File: .ci/azure/linux.yml [hunk 2]
```diff
@@ -89,17 +89,17 @@ jobs:
...
+        curl -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2019-06-01"
+        whoami
+        uname -a
+        echo Python3 info ; which python3 ; python3 --version
+        echo Python info ; which python ; python --version
+        echo Java info ; which java ; java -version
+        echo gcc info ; which gcc ; gcc --version
+        echo cmake info ; which cmake ; cmake --version
+        lsb_release
+        env
```
Comment (rkazants) [inline .ci/azure/linux.yml:102]: Not sure if we need to change this. please revert since not neccassary.

Comment (rnugmanx) [reply .ci/azure/linux.yml]: Of course, removed

File: .ci/azure/linux.yml (hunk 1 above)
Comment (rnugmanx) [reply .ci/azure/linux.yml]: Done, please, check

**Comment by rkazants** [comment]
@rnugmanx, let us anticipate green CI and merge if it is fine. In general, LGTM.

**Review by rkazants — APPROVED** [review]

**Comment by rnugmanx** [comment]
Rebuild

**Comment by rnugmanx** [comment]
@rkazants, @dkrupnov, added tests consumes ~1-1:30 min:
https://dev.azure.com/openvinoci/dldt/_build/results?buildId=396954&view=logs&j=982848f2-7f8d-5e57-730a-8df1e5f991ba&t=1e3c027f-184d-5320-0cd9-8d4d51da753f

**Pull request merged**